src/renderers/dom/shared/DOMPropertyOperations.js

```diff
@@ -161,6 +161,8 @@ var DOMPropertyOperations = {
     } else if (DOMProperty.isCustomAttribute(name)) {
       DOMPropertyOperations.setValueForAttribute(node, name, value);
       return;
+    } else if (isCustomElement) {
+      node[name] = value;
```

Comment:
Like I described above, custom properties on custom elements won't have a `propertyInfo` definition, but we still want to set them on the element.